ffi-cdecl/SDL2_0_decl.c

```diff
@@ -220,6 +220,7 @@ cdecl_const(SDL_WINDOWPOS_UNDEFINED)
 cdecl_const(SDL_WINDOW_FULLSCREEN)
 cdecl_const(SDL_WINDOW_FULLSCREEN_DESKTOP)
 cdecl_const(SDL_WINDOW_RESIZABLE)
+cdecl_const(SDL_WINDOW_ALLOW_HIGHDPI)
```

Comment:
Btw, do you want to put it in alphabetically?

Reply:
I'd actually prefer it in increasing *value* order (i.e., the same order as the header), which I'm not sure is actually the case either :D.

Reply:
That's an interesting point. RESIZABLE should go in between FULLSCREEN and FULLSCREEN_DESKTOP then. ;-)

Reply:
Fwiw I think they should go in value order too